kythe/java/com/google/devtools/kythe/platform/kzip/KZipWriter.java

```diff
@@ -94,7 +96,7 @@ private void appendZip(byte[] data, String path) throws IOException {
       output.write(data);
       output.closeEntry();
     } else {
-      System.err.println("Warning: Already wrote " + path + " to kzip.");
+      logger.atWarning().log("Warning: Already wrote " + path + " to kzip.");
```

Comment:
nit: use the built in log formatters

"Warning: already wrote %s to kzip.", path);

In this case it doesn't actually matter, but the habit is worth forming because that one time you write a logger.atInfo().log("String appended with " + someExpensiveToMarshalObject); in a performance-critical loop, you'll have a bad time :P

Reply:
done